Examples/Sample_xDscWebServiceRegistrationWithSecurityBestPractices.ps1

```diff
@@ -53,7 +53,7 @@ configuration Sample_xDscWebServiceRegistrationWithEnhancedSecurity
             DependsOn               = "[WindowsFeature]DSCServiceFeature" 
             RegistrationKeyPath     = "$env:PROGRAMFILES\WindowsPowerShell\DscService"   
             AcceptSelfSignedCertificates = $true
-            UseUpToDateSecuritySettings = $true
+            UseSecurityBestPractices = $true
```

Comment:
"This is the issue for all DSC configurations and reason to introduce inventory work?"

 Is this true? For the configuration above, if the user sets AcceptSelfSignedCertificates to $true, and then they change it to $false we will stop accepting self-signed certificates, right? However, if they set UseSecurityBestPractices to $true and then they change it to $false then we will not stop enforcing the secure protocols in the schannel.

---
In reply to: [85399745](https://github.com/PowerShell/xPSDesiredStateConfiguration/pull/261#discussion_r85399745) [](ancestors = 85399745)